frontend/components/events/create/CreateEventStepper.tsx

```diff
@@ -24,67 +24,71 @@ export default function CreateEventStepper({
     <Stepper activeStep={activeStep}>
       <Step onClick={() => {}}>
         <UserIcon className="h-5 w-5" />
-        <div className="absolute -bottom-[4.5rem] w-max text-center">
+        <div className={`absolute -bottom-[4.5rem] w-max text-center ${activeStep === 0 ? '' : 'hidden sm:flex sm:flex-col items-center'}`}>
           <Typography
             variant="h6"
             color={activeStep === 0 ? "blue-gray" : "gray"}
+            className={activeStep === 0 ? '' : 'hidden sm:inline'}
```

Comment:
if we hide the div above, why do we need to hide it in the child divs as well, line 31 and 37

Reply:
there a slight issue if we dont do this line where the stepper loses the first step